src/Helpers.fs

```diff
@@ -20,34 +20,24 @@ module VSCode =
         |> Option.map (fun e -> e.extensionPath)
 
 
+module CrossSpawn = 
```

Comment:
this does require adding the cross-spawn package, but that has to be handled on a consumer basis until/unless we package this repo as femto packages.